.stack-to-nix.cache

```diff
@@ -0,0 +1,12 @@
+https://github.com/input-output-hk/cardano-prelude 2256fd727c5f92e6218afdcf8cddf6e01c4a9dcd . 0fdwlhkpc7inkqflcdzinx9qr5g3i34clzhl6iiagj851c3jcgsn cardano-prelude cardano-prelude.nix
```

Comment:
I would remove this file from git because it often causes problems when regenerating with stack-to-nix.